app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -1,10 +1,13 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
+// import androidx.room.Entity
+// import androidx.room.PrimaryKey
```

Comment:
Remove this entirely.

Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.